qt/aqt/filtered_deck.py

```diff
@@ -100,6 +108,17 @@ def _initial_dialog_setup(self) -> None:
 
         restoreGeom(self, self.GEOMETRY_KEY)
 
+    def _after_load_deck_setup(self, deck_id) -> None:
+        form = self.form
+        re_enable = self._re_enable_allow_empty_toggle
+        if deck_id != 0:
+            qconnect(form.search.textChanged, re_enable)
+            qconnect(form.order.currentTextChanged, re_enable)
+            qconnect(form.limit.valueChanged, re_enable)
+            qconnect(form.search_2.textChanged, re_enable)
+            qconnect(form.order_2.currentTextChanged, re_enable)
+            qconnect(form.limit_2.valueChanged, re_enable)
```

Comment:
This seems more complicated than necessary. Why not just show the option all the time then, and avoid the extra code that checks if the deck is new? It should also be changed to say "Create/update this deck even if ...".

Reply:
I agree that it is too complicated, but since you asked for it to be hidden on old decks I thought I'd include it and let you make the call. I'll it change to always show and update the string.